addons/mail/static/src/owl/components/composer/composer.js

```diff
@@ -163,10 +163,11 @@ class Composer extends Component {
         // TODO: take suggested recipients into account
         try {
             await this.storeDispatch('postMessage', this.props.composerLocalId, {
-                htmlContent: this._textInputRef.comp.getHtmlContent(),
+                htmlContent: this._getTextInputContentAsHtml(this._textInputRef.comp.getContent()),
```

Comment:
move `_getTextInputContentAsHtml()` logic to the store

Reply:
Done

Reply:
Done